karrot/notifications/receivers.py

```diff
@@ -188,9 +188,20 @@ def new_member(sender, instance, created, **kwargs):
 
 @receiver(post_delete, sender=GroupMembership)
 def group_member_removed(sender, instance, **kwargs):
-    """Remove notification when leaving group"""
+    """Remove notification when leaving group."""
     membership = instance
 
+    for user in membership.group.members.all():
+        Notification.objects.create(
+            user=user,
+            type=NotificationType.MEMBER_LEFT.value,
+            context={
+                'group': membership.group_id,
+                'user': membership.user_id,
+                'added_by': membership.added_by_id,
```

Comment:
I think we don't need `added_by` here, as this is just about leaving.

Reply:
hey @nicksellen I've addressed this change, and also added the code for the time limit functionality(60 days). Let me know if that looks good :slightly_smiling_face: